### PR TITLE
add invites to policy settings

### PR DESCRIPTION
**Pull request by hobbitronics**
image

### CODE REVIEW

File: src/data/policies.ts [hunk 1]
```diff
@@ -80,11 +87,12 @@ const updatePolicyStore = (changedPolicy: Policy) => {
     if (i === -1) {
       policies.push(changedPolicy)
     } else {
-      // the policies returned from the get policies list never include claims on them
+      // the policies returned from the get policies list never include claims or invites on them
```
Comment (Schparky) [inline src/data/policies.ts:90]: Don't know if it's worth mentioning, but dependents are also omitted.

**Review by Schparky — APPROVED** [review]

File: src/data/policies.ts (hunk 1 above)
Comment (hobbitronics) [reply src/data/policies.ts:90]: and what about member?

Comment (hobbitronics) [reply src/data/policies.ts]: Added the code to keep both

Comment (Schparky) [reply src/data/policies.ts]: > and what about members?

Members come with PoliciesList.

Comment (hobbitronics) [reply src/data/policies.ts]: thanks

**Pull request merged**